tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "importHelpers": true,
     "declaration": true,
     "sourceMap": true,
-    "rootDir": "./",
+    "rootDir": "./src",
```

Comment:
Been meaning to fix that tsdx warning 🙏 Will have to move the `__tests__` dir into `src` as well, but no big deal.